guides/user/kubernetes-v2/deploy-manifest/index.md

```diff
@@ -0,0 +1,173 @@
+---
+layout: single
+title:  "Deploy Kubernetes Manifests"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+This guide is meant to show you the very basics of how to deploy a Kubernetes
+Manifest using Spinnaker, using the [Kubernetes Provider
+V2](/setup/install/providers/kubernetes-v2).
+
+There are two things to cover here:
+
+* [How to specify which manifest to deploy](#specifying-your-manifest)
+* [How to override artifacts in the manifest](#overriding-artifacts)
+
+At a high level, you specify a manifest to deploy, and optionally, can override
+certain artifacts (as fields) at runtime (e.g. the docker image to use).
+
+## Specifying Your Manifest
+
+Depending on your needs, there is more than one way to specify the manifest
+that you want to deploy:
+
+* [Statically: directly in the pipeline](#statically-specified-manifests)
+* [Dynamically: bound at runtime using an
+  artifact](#dynamically-specified-manifests)
+
+In either case, we want to start by selecting the __Deploy (Manifest)__ stage
+from the stage selector as shown here:
+
+{%
+  include
+  figure
+  image_path="./deploy-manifest.png"
+%}
+
+> :warning: The regular __Deploy__ stage is used to deploy more opinionated
+> "Server Groups" in Spinnaker, using either another provider, or the
+> Kubernetes V1 provider.
+
+### Statically specified manifests
+
+If you know ahead of time what you expect to deploy a certain manifest (even if
+you don't know what version of your docker image it will run) you can declare
+it directly in the pipeline the manifest specification.
+
+{%
+  include
+  figure
+  image_path="./in-pipeline.png"
+  caption="Notice that by selecting \"Text\" as the __Manifest Source__, we get
+  to enter the Manifest YAML by hand."
+%}
+
+Of course, if you are generating your pipeline definitions rather than entering
+them into the UI, the stage definition would look more like:
+
+```json
+{
+  "name": "Deploy my manifest",   // human-readable name
+  "type": "deployManifest",       // tells orchestration engine what to run
+  "account": "nudge",             // account (k8s cluster) to deploy to
+  "cloudProvider": "kubernetes",
+  "source": "text",
+  "manifest": {
+                                  // manifest contents go here
+  },
+  "moniker": {                    // used to specify app & cluster for grouping
+                                  // resources in UI
+    "app": "xnat",
+    "cluster": "c7",
+  }
+}
+```
+
+### Dynamically specified manifests
```

Comment:
"Specifying Manifests Dynamically"

btw, I see you're using title-caps for some headings and sentence-caps for others. I strongly prefer sentence-caps, like in this one, and spinnaker.io uses an inconsistent mix of both--mostly title-caps. So for now I'm using title caps for everything, but at somepoint I'm going to change them all to sentence-caps. A low-priority effort, but in the mean time I recommend using title-caps everywhere.